### ISSUE TITLE
Cnchi 0.14.78 - 0.14.89 error

### ISSUE THREAD

**Issue by Archernoob** [Antergos/Cnchi#605]
The first problem is with antergos-2016.05.28-x86_64.iso, the Cnchi does not start automatically and must be updated manually.
On the other hand, I have problem with Cnchi 0.14.78-0.14.82 Cinnamon installation.

[cnchi.log](https://www.zerobin.net/?b507f0f0904da830#g4VjgaYnOTNOokqkNeTpckCPRlRiOd7G0bt412S2J1U=)

**Comment by Arup65** [comment]
I have issue with cnchi as well today as installation stops after entering name with a long message of utf-8 encoding and other stuff.

**Comment by lots0logs** [comment]
This has been fixed. Cnchi package is updating as I type this. Just run `pacman-boot` command when ISO starts to update to the latest version.

**Comment by Archernoob** [comment]
Same problem again with Cnchi 0.14.85.
`error: 144563`

**Comment by devland** [comment]
Yup. Still not working. I'm getting the error reference 145380.

```
2016-05-31 22:03:54 [INFO] slides.py(246) manage_events_from_cb_queue(): Checking disk space...
2016-05-31 22:03:54 [INFO] slides.py(246) manage_events_from_cb_queue(): Downloading missing keys into the keyring...
2016-05-31 22:03:56 [INFO] slides.py(246) manage_events_from_cb_queue(): Loading packages...
2016-05-31 22:04:04 [ERROR] pac.py(155) finalize_transaction(): Can't finalize alpm transaction: transaction failed, pm_errno 34 (invalid or corrupted package), []
2016-05-31 22:04:05 [DEBUG] pac.py(157) finalize_transaction(): Releasing alpm transaction...
2016-05-31 22:04:05 [DEBUG] pac.py(159) finalize_transaction(): Alpm transaction done.
2016-05-31 22:04:05 [ERROR] process.py(128) run(): __exit__
2016-05-31 22:04:05 [ERROR] process.py(132) run(): Traceback (most recent call last):
2016-05-31 22:04:05 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/process.py", line 107, in run
    self.install_screen.run_install(self.pkg.packages, self.down.metalinks)
2016-05-31 22:04:05 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/advanced.py", line 2387, in run_install
    self.installation.start()
2016-05-31 22:04:05 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/misc/extra.py", line 191, in helper
    return func(*args, **kwargs)
2016-05-31 22:04:05 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/install.py", line 294, in start
    self.install_packages()
2016-05-31 22:04:05 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/install.py", line 449, in install_packages
    with misc.raised_privileges:
2016-05-31 22:04:05 [ERROR] process.py(132) run(): AttributeError: __exit__
2016-05-31 22:04:06 [INFO] connectionpool.py(213) _new_conn(): Starting new HTTP connection (1): build.antergos.com
2016-05-31 22:04:07 [DEBUG] connectionpool.py(395) _make_request(): "GET /hook?cnchi=2580656b5c7be34e4144a3129a95e421&install_id=145380&result=False HTTP/1.1" 301 None
2016-05-31 22:04:07 [DEBUG] connectionpool.py(395) _make_request(): "GET /api/hook?cnchi=2580656b5c7be34e4144a3129a95e421&install_id=145380&result=False HTTP/1.1" 200 None
```

**Comment by lots0logs** [comment]
The latest version is actually 0.14.88. Please try manually updating cnchi by running this command: `pacman-boot`

**Comment by devland** [comment]
pacman-boot didn't work. :(

```
[antergos@ant-16 ~]$ pacman-boot

(gnome-shell:1952): Gtk-WARNING **: Theme parsing error: gtk.css:3978:29: The style property GtkButton:inner-border is deprecated and shouldn't be used anymore. It will be removed in a future version
Gjs-Message: JS LOG: Failed to launch ibus-daemon: Failed to execute child process "ibus-daemon" (No such file or directory)
Gjs-Message: JS LOG: Failed to register AuthenticationAgent
Gjs-Message: JS LOG: Failed to add search provider /usr/share/gnome-shell/search-providers/org.gnome.bijiben-search-provider.ini: TypeError: appInfo is null
Gjs-Message: JS LOG: No permission to trigger offline updates: Polkit.Error: GDBus.Error:org.freedesktop.PolicyKit1.Error.Failed: Action org.freedesktop.packagekit.trigger-offline-update is not registered

(gnome-shell:1952): GLib-Net-WARNING **: couldn't load TLS file database: Failed to open file '/etc/ssl/certs/ca-certificates.crt': No such file or directory
Gjs-Message: JS LOG: loading user theme: /usr/share//themes/Numix-Frost/gnome-shell/gnome-shell.css
```

**Comment by lots0logs** [comment]
Did you run the command using <kbd>Alt</kbd>+<kbd>F2</kbd>?

**Comment by devland** [comment]
I managed to update cnchi to 0.14.89 but I got the same error at the end. :(
"transaction failed, pm_errno 34 (invalid or corrupted package)"

**Comment by Archernoob** [comment]
Same error again with 0.14.89.
error: 145980
Don't get me wrong, but how something can be in stable branch if we can't install system for few days? Cnchi 0.14.2x worked very well.

**Comment by bidlocode** [comment]
I'm still experiencing the issue.

**Comment by devland** [comment]
I tried with xfce. It installed correctly.
Still, cinnamon is what we really wanted. :)

**Comment by cguentherTUChemnitz** [comment]
@bidlocode: When it is known, that Cnchi 0.14.2x is working properly, why not revert to this state and apply commit by commit testing when the problem occurs.

How we are able to run the cnchi in a specific downgraded version?

**Comment by cguentherTUChemnitz** [comment]
Isn't it the same error as described in this ticket?
https://github.com/Antergos/Cnchi/issues/603

Both reports shows the log entry
> Can't finalize alpm transaction: transaction failed, pm_errno 34 (invalid or corrupted package), []

**Comment by devland** [comment]
> Isn't it the same error as described in this ticket? #603

Yup. It's the same issue.

**Comment by Archernoob** [comment]
I think that this problem is not caused by Cnchi, problem may be caused by packages on Antergos server. The same problem has already been reported https://forum.antergos.com/topic/4180/not-able-to-install-after-5-attempts.

**Comment by cguentherTUChemnitz** [comment]
I do not think that this is caused by corrupt packages. I tried:
 - Clearing package cache with `sudo pacman -Scc`, confirming all questions with yes
 - Setting up mirror for very fast connection (I am sitting inside an TU-Chemnitz building and i am using the TU-Chemnitz mirror with symmetric gigabit connection)

The problem still exists only for cinnamon. So i thought it might be possible that the mirror contains a corrupt package. So i switched to only highly updated mirrors and tried again, with the same problem.

So i wonder, if there is a corrupt package on all mirrors, why only the antergos cnchi installation suffers and not the update process of normal arch / antergos?

Additionally i see only the line `Can't finalize alpm transaction: transaction failed, pm_errno 34 (invalid or corrupted package), []`, but i do not have any hint which package is affected.

**Comment by Archernoob** [comment]
One good news, everything is fine now! :) I just successfully completed the installation with latest Cnchi. As I assumed, the problem is not in Cnchi but in packages. Please, try now and confirm this. 
Big apology for @lots0logs!

**Comment by cguentherTUChemnitz** [comment]
Thats very sad. I assume you have to be lucky to get the right packages from the right mirrors. I used `pacman-boot` command with `ALT F2` to update to lasted state. Afterwards i cleaned the pacman cache with `sudo pacman -Scc`, updated the package list with `sudo pacman -Syy` and updated the current libs with `sudo pacman -Syu`. It also updated the pamac package, but nothing seem to fix the problem. The same error occured after the downloading of the cinnamon packages. Very sad -.- So i am not able to confirm it as solved.

I will try again with a modified mirrorlist using the most recent updated mirrors as first ones.

**Comment by cguentherTUChemnitz** [comment]
@Archernoob Are you able to discribe the steps you have done? Did you some special updates in front of running the cnchi? Which image version did you use?

**Comment by Archernoob** [comment]
I noticed another interesting thing. The Cinnamon installation has passed only one time on server in Germany when I had not changed anything. I used antergos-2016.05.28-x86_64.iso and pacman-boot to update Cnchi.
I constantly use the same combination of programs and Cnchi sometimes shows 809, 808, 807 and 806 packages, even the exact number of packages is 808.
Today cinnamon is updated to 3.0.5-1 and Cnchi download from somewhere 3.0.4.x version.
Don't forget to look into antergos-mirrorlist.

**Comment by bidlocode** [comment]
Can't confirm it solved :-( it's still there.

**Comment by Archernoob** [comment]
I am writing to you from Cinnamon with solution for this problem. Comment everything in /etc/pacman.d/antergos-mirrorlist except this mirror.
`mirror.de.leaseweb.net/antergos`
The only thing I do not understand is why Cnchi wrote that he download 808 packages, and after that wrote that installing 805 packages.

**Comment by cguentherTUChemnitz** [comment]
I run `ALT F2` with `pacman-boot`

Then i cleaned the mirrorlist file and added only the up-to date ones:
I used the following arch-mirrors as the newest ones (/etc/pacman.d/mirrorlist):
Server = http://archlinux.polymorf.fr/$repo/os/$arch
Server = http://mirror.js-webcoding.de/pub/archlinux/$repo/os/$arch

Additionally i used those antergos mirrors (/etc/pacman.d/antergos-mirrorlist):
Server = http://cinnarch.polymorf.fr/$repo/$arch
Server = http://cinnarch.limun.org/$repo/$arch
Server = http://mirror.de.leaseweb.net/antergos/$repo/$arch

I updated the package list with `pacman -Syy` and run the cnchi. 

But the problem still exists -.-

**Comment by cguentherTUChemnitz** [comment]
I still can't install cinnamon. I replayed my previously steps and stripped the antergos-mirrorlist to use only the german leaseweb server. It still does not solve the alpm problem.

**Comment by Archernoob** [comment]
@cguentherTUChemnitz I just tried it again without any changes except changes in antergos-mirrorlist. I hope it does not repeat the story like with [Mint](http://blog.linuxmint.com/?p=2994).
First run `Alt+F2` with `pacman-boot`, after that disable all repo in antergos-mirrorlist except `Server = http://mirror.de.leaseweb.net/antergos/$repo/$arch` and start Cnchi. Don't run `pacman -Syy`, it is possible to make changes over `pacman-boot`.

**Comment by cguentherTUChemnitz** [comment]
@Archernoob Thanks for the detailed way to go. This is indeed a running reproducable way.

Very strange is that updating the package list with -Syy will break the process -.-

**Comment by devland** [comment]
@Archernoob It didn't work for me. I still get the error at the end.

**Comment by cguentherTUChemnitz** [comment]
> Today cinnamon is updated to 3.0.5-1 and Cnchi download from somewhere 3.0.4.x version.
The same applies for my installation. The crazy thing is, that the installation was, that the installed cinnamon knows about the possible update of cinnamon right after bootup. So it looks like some strange mirror luck to get the 3.0.4.x version and not the current 3.0.5-1 during installation with cnchi.

**Comment by devland** [comment]
Apparently they're too busy with generating paid facebooks advertisement posts. :)
https://www.facebook.com/antergos/posts/1743538295922738

**Comment by lots0logs** [comment]
@devland Cloudstats graciously provides us with completely free use of their server monitoring platform. The only thing they ask in return is that we occasionally help spread the word about their service. I don't think there is anything wrong with that.

The Cinnamon issue should be resolved. Its fixed in all my tests. The steps to take are quite simple too:

1. Boot the ISO
2. <kbd>Alt</kbd>+<kbd>F2</kbd>, `pacman-boot`, <kbd>Enter</kbd>
3. Wait for Cnchi to appear and then proceed as normal.

As soon as we confirm the original problem with gdk pixbuf query cache is resolved upstream we will build a new set of ISOs. In the meantime the process I described above is all that is required to perform a successful installation.

**Comment by devland** [comment]
Sorry if I was negative about your sponsors. It's very nice of them for helping you out. :)

I already tried alt+f2, pacman-boot to get the 0.14.89 version of cnchi.
The installation still fails right after the package loading phase with the same error mentioned above.
"Can't finalize alpm transaction: transaction failed, pm_errno 34 (invalid or corrupted package)"

Maybe I'm getting bad luck with the mirror lists.
I'll try again this evening.

Thanks again for all the help. I really like antergos. :)

**Comment by bidlocode** [comment]
>  Alt+F2, pacman-boot, Enter 

At this moment, finally, it worked for me.
Big thanks to @lots0logs and all participants of this thread.

**Comment by Archernoob** [comment]
> Alt+F2, pacman-boot, Enter

Won't work for me.
Only with antergos mirror changes installation is successful. Without changes in antergos mirrors I have no luck.
@bidlocode did you maybe noticed number of downloaded and installed packages?
@lots0logs Cnchi likely use USA mirrors in your case.
Can you explain me why Cnchi fetched 808 packages and installed only 805? Today when I tested, Cnchi fetched only 777 packages.
[cnchi.log](https://www.zerobin.net/?242fb319e8386588#FbZLOKWYrv+uomaq0L+QzVkKIZM4o5U9ik4t+mhZHnc=)

**Comment by lots0logs** [comment]
Looks like there was an issue with the mirrors. I've made some adjustments. Wait two hours and then you should be able to complete an installation. Sorry about that!

**Comment by Archernoob** [comment]
Same problem again and again... Sorry guys, I give up.

**Comment by cguentherTUChemnitz** [comment]
Maybe it is a useful information. Directly after installation, the cinnamon package was out of date (3.0.4.x  against 3.0.5-1) but all other cinnamon packages provided by the antergos repo where newer than those in the core and community mirror.

**Comment by devland** [comment]
Still not working with 0.14.89.
I've even tried editing the n=mirror lists as suggested above.
```
2016-06-02 19:03:40 [INFO] slides.py(246) manage_events_from_cb_queue(): Downloading missing keys into the keyring...
2016-06-02 19:03:41 [INFO] slides.py(246) manage_events_from_cb_queue(): Loading packages...
2016-06-02 19:03:51 [ERROR] pac.py(155) finalize_transaction(): Can't finalize alpm transaction: transaction failed, pm_errno 34 (invalid or corrupted package), []
2016-06-02 19:03:51 [DEBUG] pac.py(157) finalize_transaction(): Releasing alpm transaction...
2016-06-02 19:03:51 [DEBUG] pac.py(159) finalize_transaction(): Alpm transaction done.
2016-06-02 19:03:51 [ERROR] process.py(128) run(): "Can't install necessary packages. Cnchi can't continue."
2016-06-02 19:03:52 [ERROR] process.py(132) run(): Traceback (most recent call last):
2016-06-02 19:03:52 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/process.py", line 107, in run
    self.install_screen.run_install(self.pkg.packages, self.down.metalinks)
2016-06-02 19:03:52 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/advanced.py", line 2387, in run_install
    self.installation.start()
2016-06-02 19:03:52 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/misc/extra.py", line 191, in helper
    return func(*args, **kwargs)
2016-06-02 19:03:52 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/install.py", line 294, in start
    self.install_packages()
2016-06-02 19:03:52 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/install.py", line 460, in install_packages
    raise InstallError(txt)
2016-06-02 19:03:52 [ERROR] process.py(132) run(): misc.extra.InstallError: "Can't install necessary packages. Cnchi can't continue."
2016-06-02 19:03:52 [INFO] connectionpool.py(213) _new_conn(): Starting new HTTP connection (1): build.antergos.com
2016-06-02 19:03:53 [DEBUG] connectionpool.py(395) _make_request(): "GET /hook?cnchi=2580656b5c7be34e4144a3129a95e421&install_id=147633&result=False HTTP/1.1" 301 None
2016-06-02 19:03:54 [DEBUG] connectionpool.py(395) _make_request(): "GET /api/hook?cnchi=2580656b5c7be34e4144a3129a95e421&install_id=147633&result=False HTTP/1.1" 200 None
```

**Comment by Archernoob** [comment]
If I only once again try to install Antergos, my brain will explode.
This is the current situation with cinnamon package.
`Server = http://mirror.bjtu.edu.cn/cinnarch/$repo/$arch` is made in China and only repo with cinnamon-3.0.4-1.
`Server = http://cinnarch.limun.org/$repo/$arch` does not have any package
`Server = http://mirror.us.leaseweb.net/antergos/$repo/$arch` does not have any cinnamon packages, and same situation is with `Server = http://mirrors.acm.wpi.edu/antergos/$repo/$arch`

The rest of antergos mirrors have cinnamon-session-3.0.1-2. Arch mirrors published "older" cinnamon-session 3.0.1-1. I did not check other packages because it seems pointless. This looks like Back to the Future. :)

Can some dev tell to us what went wrong?

**Comment by Archernoob** [comment]
Same error again...

**Comment by lots0logs** [comment]
There's a bug in our build server that is preventing me from moving the latest version to the main repo. I am at work so I dont have time to look into it. The new package (v0.14.91) is in staging repo and can be installed from there if anyone wants to do that in the meantime. Sorry for the trouble!

**Comment by devland** [comment]
> The new package (v0.14.91) is in staging repo and can be installed from there

@lots0logs How do we do that?

**Comment by karasu** [comment]
https://antergos.com/wiki/system-admin/package-management/antergos-staging-repo/

**Comment by Archernoob** [comment]
The staging repo from @karasu link does not have any cnchi package, the v0.14.91 you can find on repo.antergos.info. This can be also performed manually:
`wget http://repo.antergos.info/antergos-staging/x86_64/cnchi-0.14.91-1-any.pkg.tar.xz`
`sudo pacman -U cnchi-0.14.91-1-any.pkg.tar.xz`

Now you can run Cnchi and start with installation, but there is no need to do that because the problem persists.

@lots0logs I tested v0.14.91. First they fetched 769 packages instead 808 and give same error again. :)

**Comment by lots0logs** [comment]
@Archernoob :facepalm: Could you post a link to the entire log if you still have it? Thanks.

**Comment by Archernoob** [comment]
@lots0logs you can find log here.
[cnchi.log](https://www.zerobin.net/?1c99eb7ddce397d7#x6nAG3LiVf17Eu58Lb8gXqujHo5bunwIt5rxT8g/1ZE=)

**Comment by Samhamsam** [comment]
I also have problems to install antergos KDE in German on my disk. It fails with Error 149008 and Cnchi 0.14.89.
Logfile:
 https://www.zerobin.net/?618494ab29891f37#701jMWMK6x1d0RnsOpIfBtqrlQyIsYIwM8OP3Q6H0c8=

**Comment by Archernoob** [comment]
@Samhamsam same issue like with Cinnamon. You can not do anything except to try with v0.14.91 as stated in previous comments.

**Comment by Samhamsam** [comment]
This time it works with Cnchi v0.14.91. But it cant boot from the disk.
Logfile:
https://www.zerobin.net/?d886b2b909c6348f#dd8vQ7+K1BPDX054icDqVsK9lKUXV2k/rWqKeUkTf+o=

**Comment by devland** [comment]
It didn't work with 0.14.91 either. :(

```
2016-06-03 23:13:53 [INFO] slides.py(246) manage_events_from_cb_queue(): Checking disk space...
2016-06-03 23:13:53 [INFO] slides.py(246) manage_events_from_cb_queue(): Downloading missing keys into the keyring...
2016-06-03 23:13:54 [INFO] slides.py(246) manage_events_from_cb_queue(): Loading packages...
2016-06-03 23:14:04 [ERROR] pac.py(155) finalize_transaction(): Can't finalize alpm transaction: transaction failed, pm_errno 34 (invalid or corrupted package), []
2016-06-03 23:14:04 [DEBUG] pac.py(157) finalize_transaction(): Releasing alpm transaction...
2016-06-03 23:14:04 [DEBUG] pac.py(159) finalize_transaction(): Alpm transaction done.
2016-06-03 23:14:04 [ERROR] process.py(128) run(): __exit__
2016-06-03 23:14:04 [ERROR] process.py(132) run(): Traceback (most recent call last):
2016-06-03 23:14:04 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/process.py", line 107, in run
    self.install_screen.run_install(self.pkg.packages, self.down.metalinks)
2016-06-03 23:14:04 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/advanced.py", line 2387, in run_install
    self.installation.start()
2016-06-03 23:14:04 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/misc/extra.py", line 191, in helper
    return func(*args, **kwargs)
2016-06-03 23:14:04 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/install.py", line 294, in start
    self.install_packages()
2016-06-03 23:14:04 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/install.py", line 450, in install_packages
    with misc.raised_privileges:
2016-06-03 23:14:04 [ERROR] process.py(132) run(): AttributeError: __exit__
2016-06-03 23:14:05 [INFO] connectionpool.py(213) _new_conn(): Starting new HTTP connection (1): build.antergos.com
2016-06-03 23:14:05 [DEBUG] connectionpool.py(395) _make_request(): "GET /hook?cnchi=2580656b5c7be34e4144a3129a95e421&install_id=149079&result=False HTTP/1.1" 301 None
2016-06-03 23:14:06 [DEBUG] connectionpool.py(395) _make_request(): "GET /api/hook?cnchi=2580656b5c7be34e4144a3129a95e421&install_id=149079&result=False HTTP/1.1" 200 None
```

**Comment by Archernoob** [comment]
@Samhamsam that's excellent. You can use https://forum.antergos.com/ for booting problem.

**Comment by Archernoob** [comment]
@lots0logs nothing new with v0.14.92. :)
[cnchi.log](https://www.zerobin.net/?25a75a7252598fbe#aMUReeh6T9U4LBTf1QXVvDceJ300fazxbIRGAkojah4=)

**Comment by devland** [comment]
Still not working on 0.14.92.

```
2016-06-04 08:34:27 [INFO] slides.py(246) manage_events_from_cb_queue(): Downloading missing keys into the keyring...
2016-06-04 08:34:27 [INFO] slides.py(246) manage_events_from_cb_queue(): Loading packages...
2016-06-04 08:34:38 [ERROR] pac.py(155) finalize_transaction(): Can't finalize alpm transaction: transaction failed, pm_errno 34 (invalid or corrupted package), []
2016-06-04 08:34:39 [DEBUG] pac.py(157) finalize_transaction(): Releasing alpm transaction...
2016-06-04 08:34:39 [DEBUG] pac.py(159) finalize_transaction(): Alpm transaction done.
2016-06-04 08:34:39 [ERROR] process.py(128) run(): [Errno 28] No space left on device
2016-06-04 08:34:39 [ERROR] process.py(132) run(): OSError: [Errno 28] No space left on device
2016-06-04 08:34:39 [ERROR] process.py(132) run(): 
During handling of the above exception, another exception occurred:
2016-06-04 08:34:39 [ERROR] process.py(132) run(): Traceback (most recent call last):
2016-06-04 08:34:39 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/process.py", line 107, in run
    self.install_screen.run_install(self.pkg.packages, self.down.metalinks)
2016-06-04 08:34:39 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/advanced.py", line 2387, in run_install
    self.installation.start()
2016-06-04 08:34:39 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/misc/extra.py", line 191, in helper
    return func(*args, **kwargs)
2016-06-04 08:34:39 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/install.py", line 294, in start
    self.install_packages()
2016-06-04 08:34:39 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/install.py", line 470, in install_packages
    new_pacman_conf.write(line)
2016-06-04 08:34:39 [ERROR] process.py(132) run(): OSError: [Errno 28] No space left on device
2016-06-04 08:34:40 [INFO] connectionpool.py(213) _new_conn(): Starting new HTTP connection (1): build.antergos.com
2016-06-04 08:34:40 [DEBUG] connectionpool.py(395) _make_request(): "GET /hook?cnchi=2580656b5c7be34e4144a3129a95e421&install_id=149448&result=False HTTP/1.1" 301 None
2016-06-04 08:34:40 [DEBUG] connectionpool.py(395) _make_request(): "GET /api/hook?cnchi=2580656b5c7be34e4144a3129a95e421&install_id=149448&result=False HTTP/1.1" 200 None
```

**Comment by lots0logs** [comment]
@devland Hmm..this is something else. See: `run(): OSError: [Errno 28] No space left on device` What size did you make your root partition?

**Comment by Samhamsam** [comment]
still not working with 0.14.92. KDE german.

First try: With feature selection
```
2016-06-04 08:32:49 [INFO] slides.py(246) manage_events_from_cb_queue(): Abrufen von libxrandr 1.5.0-1 (246/901)...
2016-06-04 08:32:49 [INFO] slides.py(246) manage_events_from_cb_queue(): Abrufen von adwaita-icon-theme 3.20-2 (247/901)...
2016-06-04 08:32:49 [ERROR] process.py(128) run(): [Errno 5] Eingabe-/Ausgabefehler
2016-06-04 08:32:49 [ERROR] process.py(132) run(): Traceback (most recent call last):
2016-06-04 08:32:49 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/process.py", line 107, in run
    self.install_screen.run_install(self.pkg.packages, self.down.metalinks)
2016-06-04 08:32:49 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/advanced.py", line 2387, in run_install
    self.installation.start()
2016-06-04 08:32:49 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/misc/extra.py", line 191, in helper
    return func(*args, **kwargs)
2016-06-04 08:32:49 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/install.py", line 288, in start
    self.download_packages()
2016-06-04 08:32:49 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/install.py", line 347, in download_packages
    download_packages.start(self.metalinks)
2016-06-04 08:32:49 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/download/download.py", line 100, in start
    if not download.start(self.metalinks):
2016-06-04 08:32:49 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/download/download_requests.py", line 170, in start
    self.is_hash_ok(path=dst_xz_cache_path, element=element)):
2016-06-04 08:32:49 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/download/download_requests.py", line 111, in is_hash_ok
    if md5hash != get_md5(path):
2016-06-04 08:32:49 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/download/download_requests.py", line 48, in get_md5
    for line in myfile:
2016-06-04 08:32:49 [ERROR] process.py(132) run(): OSError: [Errno 5] Eingabe-/Ausgabefehler
2016-06-04 08:32:50 [INFO] connectionpool.py(213) _new_conn(): Starting new HTTP connection (1): build.antergos.com
2016-06-04 08:32:50 [DEBUG] connectionpool.py(395) _make_request(): "GET /hook?cnchi=2580656b5c7be34e4144a3129a95e421&install_id=149464&result=False HTTP/1.1" 301 None
2016-06-04 08:32:51 [DEBUG] connectionpool.py(395) _make_request(): "GET /api/hook?cnchi=2580656b5c7be34e4144a3129a95e421&install_id=149464&result=False HTTP/1.1" 200 None
```
Second try: Without feature selection
```
2016-06-04 09:30:50 [DEBUG] pac.py(157) finalize_transaction(): Releasing alpm transaction...
2016-06-04 09:30:50 [DEBUG] pac.py(159) finalize_transaction(): Alpm transaction done.
2016-06-04 09:30:50 [ERROR] process.py(128) run(): 'Installation notwendiger Pakete nicht möglich. Cnchi kann nicht fortgesetzt werden.'
2016-06-04 09:30:50 [ERROR] process.py(132) run(): Traceback (most recent call last):
2016-06-04 09:30:50 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/process.py", line 107, in run
    self.install_screen.run_install(self.pkg.packages, self.down.metalinks)
2016-06-04 09:30:50 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/advanced.py", line 2387, in run_install
    self.installation.start()
2016-06-04 09:30:50 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/misc/extra.py", line 191, in helper
    return func(*args, **kwargs)
2016-06-04 09:30:50 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/install.py", line 294, in start
    self.install_packages()
2016-06-04 09:30:50 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/install.py", line 478, in install_packages
    raise InstallError(txt)
2016-06-04 09:30:50 [ERROR] process.py(132) run(): misc.extra.InstallError: 'Installation notwendiger Pakete nicht möglich. Cnchi kann nicht fortgesetzt werden.'
2016-06-04 09:30:51 [INFO] connectionpool.py(213) _new_conn(): Starting new HTTP connection (1): build.antergos.com
2016-06-04 09:30:51 [DEBUG] connectionpool.py(395) _make_request(): "GET /hook?cnchi=2580656b5c7be34e4144a3129a95e421&install_id=149491&result=False HTTP/1.1" 301 None
2016-06-04 09:30:51 [DEBUG] connectionpool.py(395) _make_request(): "GET /api/hook?cnchi=2580656b5c7be34e4144a3129a95e421&install_id=149491&result=False HTTP/1.1" 200 None
```

**Comment by Archernoob** [comment]
@lots0logs nothing new with v0.14.93.
[cnchi.log](https://www.zerobin.net/?fda7a00c92ec5710#NmsEP9Kh15+P4BnlOEB2eGN/I4Y6MNZWkVg8c/5wivQ=)

**Comment by Archernoob** [comment]
I'm not a Python programmer and do not want to offend anyone, but commits in the last few days with which are attempting to fix this issue, looks more like car repair with scotch tape. Today, after almost a whole week, we had total packages chaos on mirrors and next to Cinnamon we can not install KDE too.
Why you don't stop antbs and make out of use until you fix problem?

**Comment by devland** [comment]
I am now writing this from a fresh install of antergos + cinnamon. :)
I managed to get the installer working by running it as root from console.
I used the pacman-root retrieved 0.14.90 version.
Hopefully this helps in resolving the issue for everyone. :)

**Comment by Archernoob** [comment]
@devland it is good to hear but I guess that you're just lucky! :) Today Antergos team completed some changes with commits and also build.antergos.com was down a period of time. I'll wait for morning to check commits and build.antergos.com logs and then i will try to confirm your success with Cinnamon.

**Comment by devland** [comment]
I just reinstalled antergos + cinnamon via cnchi 0.14.72.
You need to run it as root. :)

**Comment by Archernoob** [comment]
I just installed Cinnamon with v0.14.94 few times and everything is ok.
@devland you don't need to use older version of Cnchi, the Cnchi v0.14.94 is now in main repo and `pacman-boot` will use it. Run it with root there are nothing with this issue, they solved the problem with antbs.
Agony came to an end and I think that this issue can be closed. :) Thank to all for help especially to @lots0logs!

**Comment by iachopolo** [comment]
I'm having the same error with cnchi 0.14.94, 0.14.111 and 0.14.113. ISO comes with 0.14.94, yesterday it updated to 0.14.111 and later to 0.14.113. I tried to install the OS with the 3 of them.

```
2016-07-19 14:05:20 [INFO] slides.py(246) manage_events_from_cb_queue(): Descargando las claves que faltan al llavero…
2016-07-19 14:05:20 [INFO] slides.py(246) manage_events_from_cb_queue(): Cargando paquetes…
2016-07-19 14:05:41 [ERROR] pac.py(157) finalize_transaction(): Can't finalize alpm transaction: transaction failed, pm_errno 34 (paquete no válido o$
2016-07-19 14:05:41 [DEBUG] pac.py(159) finalize_transaction(): Releasing alpm transaction...
2016-07-19 14:05:41 [DEBUG] pac.py(161) finalize_transaction(): Alpm transaction done.
2016-07-19 14:05:41 [ERROR] process.py(128) run(): 'No se pueden instalar los paquetes necesarios. Cnchi no puede continuar.'
2016-07-19 14:05:41 [ERROR] process.py(132) run(): Traceback (most recent call last):
2016-07-19 14:05:41 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/process.py", line 107, in run
    self.install_screen.run_install(self.pkg.packages, self.down.metalinks)
2016-07-19 14:05:41 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/automatic.py", line 360, in run_install
    self.installation.start()
2016-07-19 14:05:41 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/misc/extra.py", line 191, in helper
    return func(*args, **kwargs)
2016-07-19 14:05:41 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/install.py", line 294, in start
    self.install_packages()
2016-07-19 14:05:41 [ERROR] process.py(132) run():   File "/usr/share/cnchi/cnchi/installation/install.py", line 482, in install_packages
    raise InstallError(txt)
2016-07-19 14:05:41 [ERROR] process.py(132) run(): misc.extra.InstallError: 'No se pueden instalar los paquetes necesarios. Cnchi no puede continuar.'
2016-07-19 14:05:42 [INFO] connectionpool.py(214) _new_conn(): Starting new HTTP connection (1): build.antergos.com
2016-07-19 14:05:48 [DEBUG] connectionpool.py(401) _make_request(): "GET /hook?cnchi=2580656b5c7be34e4144a3129a95e421&install_id=204232&result=False $
2016-07-19 14:05:49 [DEBUG] connectionpool.py(401) _make_request(): "GET /api/hook?cnchi=2580656b5c7be34e4144a3129a95e421&install_id=204232&result=Fa$
```

**Comment by iachopolo** [comment]
I know the issue is closed, but it was exactly the same error so I think I should comment here...

**Comment by lots0logs** [comment]
@iachopolo Could you post a [paste2](http://paste2.org) link to the full log?

**Comment by iachopolo** [comment]
[cnchi.log](http://paste2.org/dgVIHBpc)
There you go :)

**Comment by lots0logs** [comment]
Please try using one of these ISOs instead: http://build.antergos.com/repo/browse/testing

**Comment by iachopolo** [comment]
Didn't work... (I tried twice) here's the log
[cnchi.log](http://paste2.org/OeekEU04)

**Comment by lots0logs** [comment]
Could you try it without enabling the libreoffice feature? I think there is a problem with those packages.

**Comment by iachopolo** [comment]
Still not working
[cnchi.log](http://paste2.org/gzdPePz0)

**Comment by lots0logs** [comment]
Okay. Its definitely one of the features you are enabling that brings with it a package conflict. I dont have time to narrow it down right now. I was able to do a successful install of Cinnamon with only the default features enabled. If you dont want to wait until we have time to track down the issue then you can install it without selecting all the optional features and it will work. Cheers!

**Comment by iachopolo** [comment]
I'll wait for you to resolve it and then try again. I had to install
another OS because I needed my computer.
Thanks anyway!!

**Comment by nikhilsable** [comment]
Hi! Not sure if anyone is still facing the issue but here is what worked for me (issue seems to be a package conflict):-

 - use the latest cnchi version with the ISO (i used Cnchi 0.14.195)
- DO NOT select any optional packages(ex. flash, windows SMB, etc - you can do that later).
- continue with install and it should be buttery smooth.

Note: I had the same package list error and decided to use defaul extra packages during install and 
it worked!

Thank you for the other folks on here for all the help.

**Comment by karasu** [comment]
This should fix the flash error: 
https://github.com/Antergos/Cnchi/commit/ce521832dfd137843f0b0bdd4f80c38070c69102

**Comment by awnumar** [comment]
I'm having the exact same issue, about a year later. Logfile: http://termbin.com/lg38v

**Comment by karasu** [comment]
Hi, I'm positive it's not the same issue (this one was related to an error in package naming, and yours seems to be a corrupted package signature). I'll test a new Cinnamon installation this afternoon.

Could you please @awnumar open a new issue with your problem and log?

Thanks!

**Comment by awnumar** [comment]
@karasu I opened one at #799. I did manage to fix the issue but I have no idea what did it or even if it was something I did that fixed it. 

I've left that issue open though just in case someone else has the same issue (and someone does), because it didn't work for me for a couple of days.
